webui/react/src/shared/configs/craco.config.js

```diff
@@ -95,6 +96,25 @@ module.exports = {
         ],
         languages: ['markdown', 'yaml', 'python'],
       }),
+      new CspHtmlWebpackPlugin(
+        {
+          'frame-src': "'none'",
+          'object-src': "'none'",
+          'script-src': "'self' cdn.segment.com",
+          'style-src': "'self' 'unsafe-inline'",
+        },
+        {
+          enabled: true,
+          hashEnabled: {
+            'script-src': true,
+            'style-src': false,
+          },
+          nonceEnabled: {
+            'script-src': false,
+            'style-src': false,
```

Comment:
Im not familiar with CSP very much, so my question could be stupid.
What made you choose `false`? In my understanding, `true` is more secure, right?

Reply:
CSP can add a hash of the script to the meta tag (basically saying, this is a script which I expected to be here) or a nonce generated for each request (https://content-security-policy.com/nonce/).  I wanted to do only one of the methods.